cassiopeia/solver/DistanceSolver.py

```diff
@@ -7,14 +7,14 @@
 """
 import abc
 import cassiopeia
+from cassiopeia.data import CassiopeiaTree
+from cassiopeia.mixins import DistanceSolverError
+from cassiopeia.solver import CassiopeiaSolver, solver_utilities
 import networkx as nx
 import numpy as np
 import pandas as pd
 from typing import Callable, Dict, List, Optional, Tuple
```

Comment:
```
import abc
from typing import Callable, Dict, List, Optional, Tuple

import networkx as nx
import numpy as np
import pandas as pd

from cassiopeia.data import CassiopeiaTree
from cassiopeia.mixins import DistanceSolverError
from cassiopeia.solver import CassiopeiaSolver, solver_utilities
```

Reply:
Hey Kevin and Ivan, mind doing the imports like this? Essentially, we place the "from" statements afterwards, except for the imports that are needed for "basic functions" like typing. I also removed the extra import of `cassiopeia` cause I realize it's unused